lib/api/expect/element.js

```diff
@@ -58,7 +58,7 @@ class ExpectElement extends BaseExpect {
   locateElement() {
     const {element} = this;
 
-    return this.elementLocator.findElement({element})
+    return this.elementLocator.findElement({element, transportAction: 'locateMultipleElements', returnSingleElement: true})
```

Comment:
we should keep this to use the original `/element` endpoint.

Reply:
It did not use the `element` endpoint to begin with because of the defaults in the `findElement` function. 
See my other comment https://github.com/nightwatchjs/nightwatch/pull/2589#pullrequestreview-617034310 for more details

Reply:
Ah yes, I thought it's `protocol/element.js`.